pkgs/dartpad_shared/lib/simple_items.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// The IP address of the localhost.
+/// For some reasons just `localhost` does not work for
+/// unit testing of ui against local backend.
+const localhostIp = '127.0.0.1';
```

Comment:
nit: rename this file to "constants.dart", "items" is a little ambiguous.

Reply:
My experience is saying that in future we may have tiny functions and/or enums, that we will want to put here. But for now 'constants' os ok.